week-3/quote-generator/quotes.js

```diff
@@ -20,6 +20,10 @@ function pickFromArray(choices) {
   return choices[Math.floor(Math.random() * choices.length)];
 }
 
+function getRandomQuote() {
+  return pickFromArray(quotes);
+}
+
```

Comment:
It doesn't hurt of course but we don't really need this wrapper function. We can just use `pickFromArray(quotes)` where we need it.